ddl/ddl_worker_test.go

```diff
@@ -165,7 +165,7 @@ func TestParallelDDL(t *testing.T) {
 		/     7		/	 	2			/		3		/	drop column	 /
 		/     8		/	 	2			/		3		/	rebase autoID/
 		/     9		/	 	1			/		1		/	add index	 /
-		/     10	/	 	2			/		null   	/	drop schema  /
+		/     10	/	 	2			/		null   	/	drop table  /
```

Comment:
I think it would miss the case that `drop schema`, we should try to refine the error handling instead of modifying the test case.

Reply:
And I think it may be a bug, we should modify the logic of getting the DDL job. We shouldn't run the drop schema job if rebase id(on the same database) isn't finished.

Reply:
Not only the rebase id, some other operations such as drop column also depends it

Reply:
Yeah.

Reply:
> And I think it may be a bug, we should modify the logic of getting the DDL job. We shouldn't run the drop schema job if rebase id(on the same database) isn't finished.

I do not think it is a bug, it is just that the `rebase id` is picked after the `drop schema` is finished. It will always happen in the real world, fixing it isn't significant.